src/main/java/us/icarenow/ApplicationCnsas.java

```diff
@@ -0,0 +1,11 @@
+package us.icarenow;
+
+import org.springframework.boot.SpringApplication;
```

Comment:
the root package for all the packages & classes should be us.icarenow.cnsas

Reply:
done